natlas-server/app/api/prepare_work.py

```diff
@@ -65,15 +64,14 @@ def prepare_work(reason: str, target: str) -> AgentWork:
     config = db.session.get(AgentConfig, 1)
     if not config:
         raise RuntimeError("We have no agent config! What happened?!")
-
+    config_dict = config.as_dict()
+    config_dict.update(scripts=",".join(config.scripts))
```

Comment:
I don't love this, I'd rather just send the list of strings to the agent and have it construct the string. But for now this keeps the changes entirely in natlas-server.